apps/omg_conformance/test/support/conformance/signatures_hashes.ex

```diff
@@ -106,8 +107,9 @@ defmodule Support.Conformance.SignaturesHashes do
   # NOTE: `solidity_hash/2` returns something like `{:ok, <<8, 195, 121, 160, 0, 0, 0, more zeroes...>>}`, on contract
   #       revert, when using `:geth` Ethereum node. If an assertion fails with such a result, it indicates the contract
   #       rejected some transaction to signhash unexpectedly.
-  defp solidity_hash(encoded_tx, contract) when is_binary(encoded_tx),
-    do: Eth.call_contract(contract, "hashTx(address,bytes)", [contract, encoded_tx], [{:bytes, 32}])
+  defp solidity_hash(encoded_tx, contract) when is_binary(encoded_tx) do
+    call_contract(contract, "hashTx(address,bytes)", [contract, encoded_tx], [{:bytes, 32}])
```

Comment:
similarly as above, this is the only use case for calling hashTx, hence, we don't want to put this in the deliverable code